bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -769,7 +771,7 @@ void PreProcessor::handlePreProcessedReqByNonPrimary(
   sendMsg(replyMsg->body(), myReplica_.currentPrimary(), replyMsg->type(), replyMsg->size());
   LOG_INFO(logger(),
            "Pre-processing completed by a non-primary replica"
-               << KVLOG(reqSeqNum, cid, reqRetryId, myReplica_.currentPrimary()));
+               << KVLOG(reqSeqNum, clientId, cid, reqRetryId, myReplica_.currentPrimary()));
```

Comment:
As per the logs clientId comes after cid can we maintain the uniformility
Start request processing by a non-primary replica reqSeqNum: 6767699002902183936, cid: 8c8cc25e-f736-4cdd-aef4-128794a6b8af, clientId: 23, senderId: 23, reqTimeoutMilli: 30000

Reply:
Done